### PR TITLE
fix(server): code.py as a filename overshadows an internal python file

### PR DESCRIPTION
**Pull request by ntindle**
### Background

Python uses `code.py` internally and debugpy throws a fit when you try and reuse it
### Changes 🏗️

renames `code.py` to `some_code.py` and updates the `pyproject.toml`

### PR Quality Scorecard ✨

- [x] Have you used the PR description template? &ensp; `+2 pts`
- [x] Is your pull request atomic, focusing on a single change? &ensp; `+5 pts`
- [ ] Have you linked the GitHub issue(s) that this PR addresses? &ensp; `+5 pts`
- [ ] Have you documented your changes clearly and comprehensively? &ensp; `+5 pts`
- [ ] Have you changed or added a feature? &ensp; `-4 pts`
  - [ ] Have you added/updated corresponding documentation? &ensp; `+4 pts`
  - [ ] Have you added/updated corresponding integration tests? &ensp; `+5 pts`
- [ ] Have you changed the behavior of AutoGPT? &ensp; `-5 pts`
  - [ ] Have you also run `agbenchmark` to verify that these changes do not regress performance? &ensp; `+10 pts`

### CODE REVIEW

**Comment by ntindle** [comment]
Issues fixed by #7498 #7497

File: rnd/autogpt_server/pyproject.toml [hunk 1]
```diff
@@ -53,8 +53,8 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry.scripts]
 app = "autogpt_server.app:main"
 cli = "autogpt_server.cli:main"
-format = "code:format"
-lint = "code:lint"
+format = "some_code:format"
```
Comment (majdyz) [inline rnd/autogpt_server/pyproject.toml]: Can we rename it anything other than some_code ?
code_quality.py, code_formatter.py, lint.py

Comment (ntindle) [reply rnd/autogpt_server/pyproject.toml]: Happy to, just was in my stack for the night to merge. We could forgo this file altogether using Poe the poet (see the build scripts in the pyproject.toml)

Comment (majdyz) [reply rnd/autogpt_server/pyproject.toml]: Renamed this to linter.py

**Review by majdyz — APPROVED** [review]

**Pull request merged**